.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.stylelintrc.json
```

Comment:
- [ ] Please note that the `.stylelintrc.json` file is not supposed to be in the gitignore file. Kindly remove it 👍🏽